doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -44,6 +44,9 @@ The tactics presented here specialize :tacn:`apply` and
    typically used to split conjunctions in the conclusion such as `A /\\ B` into
    two new goals `A` and `B`.
 
+   .. todo: split does more than what's described.  For example, this works:
+      Goal True. Proof. split. Qed.
```

Comment:
No, the description is correct. `True` is an inductive type with a single constructor, which does not take any argument. Therefore, `constructor 1` does solve the goal and so does `split`. The description is also correct that this is not the typical use case.